e2e/hook/config.go

```diff
@@ -0,0 +1,25 @@
+package hook
+
+import (
+	"os"
+
+	"k8s.io/client-go/rest"
+)
+
+const envVarUseInsecure string = "E2E_USE_INSECURE_TLS"
+
+func setTLSFromEnv(cfg *rest.Config) {
+	cfg.Insecure = os.Getenv(envVarUseInsecure) != "false"
```

Comment:
IIRC, if cfg.Insecure is set to true and `CAFile` or `CAData` are also set, we'll get an error

Reply:
I don't see that reflected in the docs for [TLSClientConfig](https://pkg.go.dev/k8s.io/client-go@v0.30.0/rest#TLSClientConfig).  Do you have a source for that?

Reply:
PTAL: https://gist.github.com/filariow/631546cba9ea57fe93e605473be78bf5

Reply:
Fixed in c72a3ce.